pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/table/PulsarDynamicTableSink.java

```diff
@@ -194,6 +195,8 @@ public SinkRuntimeProvider getSinkRuntimeProvider(Context context) {
         // check if metadata is used at all
         final boolean hasMetadata = metadataKeys.size() > 0;
 
+        final long delayMilliseconds = Long.valueOf((String) Optional.ofNullable(this.properties.get(PulsarOptions.SEND_DELAY_MILLISECONDS)).orElse(0));
```

Comment:
There is a type conversion error here, which needs to be fixed. 

```
java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
2021-06-11T02:37:17.2330044Z 	at org.apache.flink.streaming.connectors.pulsar.table.PulsarDynamicTableSink.createPulsarSerializer(PulsarDynamicTableSink.java:198)
2021-06-11T02:37:17.2339036Z 	at org.apache.flink.streaming.connectors.pulsar.table.PulsarDynamicTableSink.getSinkRuntimeProvider(PulsarDynamicTableSink.java:162)
2021-06-11T02:37:17.2345785Z 	at org.apache.flink.table.planner.plan.nodes.common.CommonPhysicalSink.createSinkTransformation(CommonPhysicalSink.scala:77)
2021-06-11T02:37:17.2356754Z 
```

example code:
```
Optional.ofNullable(this.properties.getProperty(PulsarOptions.SEND_DELAY_MILLISECONDS, "0"))
                .filter(StringUtils::isNumeric)
                .map(Long::valueOf)
                .orElse(0L);
```

Reply:
sorry, I fixed it.